PostHog/Error Tracking/Models/PostHogBinaryImageInfo.swift

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
lets rename the package from `Error Tracking` without an extra space, i've seen issues with that